app/controllers/application_controller.rb

```diff
@@ -1,2 +1,22 @@
 class ApplicationController < ActionController::API
+  before_action :authenticate_user
+
+  attr_accessor :current_user
+
+  private
+  def authenticate_user
+    begin
+      token = request.headers['Authorization']
+      user = User.find_by!(authentication_token: token)
```

Comment:
create a scope for active and use `User.active.find_by!(authentication_token: token)`